internal/k8s/controllers/config_controller.go

```diff
@@ -129,6 +136,7 @@ func (r *ConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 func (r *ConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&metallbv1beta2.BGPPeer{}).
+		Watches(&source.Kind{Type: &corev1.Node{}}, &handler.EnqueueRequestForObject{}).
```

Comment:
Yes, for the speaker. There are two different callbacks, one for nodes only and one for config that have different purpouses.
On the controller, I think we can remove the node handler because the config handler eventually converges in syncpeers.

Reply:
Actually, I tried and we _could_ get rid of it by calling node handler in the config reconciler.
However, it feels cleaner to have one reconciler per handler, and the node handler has a different purpouse from the confighandler (setting the labels in the bgp protocol), so unless we add a "config + node handler" where we do both (which seems super ugly) I think it's more clear to leave it as it is unless we realize it brings much performance penalty.